### PR TITLE
TC_004 and TC_005 in YakuninaTest class were updated

### PR DESCRIPTION
**Pull request by Natalia-Java-QA**
1. TC_004 and TC_005 were updated, implicitlyWait was removed from tests in YakuninaTest class

2. A closing curvy bracket was added to Inna Kushchak test, as without it I couldn't check if my updated tests are working

### CODE REVIEW

File: src/test/java/InnaKushchakTest.java [hunk 1]
```diff
@@ -34,63 +34,64 @@ public void testVerifyingLogInButton() {
...
 
-        WebElement featureProducts=getDriver().findElement(By.xpath("//div[@class='wp-block-group alignfull']//ul"));
-        List<WebElement> products=featureProducts.findElements(By.tagName("li"));
-        for (WebElement elem:products) {
-         List<WebElement> x= elem.findElements(By.tagName("h2"));
+        WebElement featureProducts = getDriver().findElement(By.xpath("//div[@class='wp-block-group alignfull']//ul"));
+        List<WebElement> products = featureProducts.findElements(By.tagName("li"));
+        for (WebElement elem : products) {
+            List<WebElement> x = elem.findElements(By.tagName("h2"));
             System.out.println(x.size());
```
Comment (tymocoder) [inline src/test/java/InnaKushchakTest.java:72]: Line 72. It is not assertion, just printing. Need to fix

File: src/test/java/InnaKushchakTest.java [hunk 2]
```diff
@@ -34,63 +34,64 @@ public void testVerifyingLogInButton() {
...
 
-        WebElement featureProducts=getDriver().findElement(By.xpath("//div[@class='wp-block-group alignfull']//ul"));
-        List<WebElement> products=featureProducts.findElements(By.tagName("li"));
+        WebElement featureProducts = getDriver().findElement(By.xpath("//div[@class='wp-block-group alignfull']//ul"));
+        List<WebElement> products = featureProducts.findElements(By.tagName("li"));
         products.get(0).getAttribute("baseURI");
         products.get(0).findElement(By.tagName("h2")).getAttribute("textContent");
         products.get(0).findElement(By.tagName("h2")).getAttribute("outerHTML");
-        System.out.println( products.get(0).findElement(By.tagName("h2")).getAttribute("innerHTML"));
+        System.out.println(products.get(0).findElement(By.tagName("h2")).getAttribute("innerHTML"));
```
Comment (tymocoder) [inline src/test/java/InnaKushchakTest.java:93]: line 93. It is not assertion, just printing. Need to fix

File: src/test/java/InnaKushchakTest.java [hunk 3]
```diff
@@ -34,63 +34,64 @@ public void testVerifyingLogInButton() {
...
+
     @Test
-    public void testBestSellers(){
-            WebElement bestSellers=getDriver().findElement(By.xpath("//ul[@class='product_list_widget']"));
-            List<WebElement> validateProducts= bestSellers.findElements(By.tagName("li"));
-            for(WebElement listOfBest:validateProducts){
+    public void testBestSellers() {
+        WebElement bestSellers = getDriver().findElement(By.xpath("//ul[@class='product_list_widget']"));
+        List<WebElement> validateProducts = bestSellers.findElements(By.tagName("li"));
+        for (WebElement listOfBest : validateProducts) {
```
Comment (tymocoder) [inline src/test/java/InnaKushchakTest.java:80]: Line 81. For loop empty

**Comment by Natalia-Java-QA** [comment]
@InnaKushchak please fix your tests.

**Pull request closed without merging**